docs/sql-ref-syntax-qry-explain.md

```diff
@@ -19,4 +19,126 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `EXPLAIN` statement provides the execution plan for the statement. 
+By default, `EXPLAIN` provides information about the physical plan.
+`EXPLAIN` does not support `DESCRIBE TABLE` statement.
+
+
+### Syntax
+{% highlight sql %}
+EXPLAIN [EXTENDED | CODEGEN | COST | FORMATTED] statement
+{% endhighlight %}
+
+### Parameters
+
+<dl>
+  <dt><code><em>EXTENDED</em></code></dt>
+  <dd>Generates Parsed Logical Plan, Analyzed Logical Plan, Optimized Logical Plan and Physical Plan.
+   Parsed Logical plan is a unresolved plan that extracted from the query.
+   Analyzed logical plans transforms which translates UnresolvedAttribute and UnresolvedRelation into fully typed objects.
+   The optimized logical plan transforms through a set of optimization rules, resulting in the Physical plan.
+  </dd>
+</dl> 
+
+<dl>
+  <dt><code><em>CODEGEN</em></code></dt>
+  <dd>Generates code for the statement, if any and a Physical Plan.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>COST</em></code></dt>
+  <dd>If plan node statistics are available, generates a logical plan and the statistics.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>FORMATTED</em></code></dt>
+  <dd>Generates two sections: a physical plan outline and node details.</dd>
+</dl>
```

Comment:
Where's a description about `input_statement`? See: https://github.com/apache/spark/blob/master/docs/sql-ref-syntax-aux-describe-query.md#parameters

Reply:
@maropu  i have written a generalised explanation for statement, as i could be any SQL query, Listing may not be feasible. Could you review if the below explanation is ok?

Reply:
How about just `Specifies a SQL statement to be explained.`